tests/dbapi20.py

```diff
@@ -554,7 +554,7 @@ def _populate(self):
             tests.
         '''
         populate = [
-            "insert into %sbooze values ('%s')" % (self.table_prefix,s)
```

Comment:
This is not a set of changes I would like to merge. You are simply using a linter that we don't use and we don't have reasons to introduce it in our codebase. So I prefer to close this MR here.